app/move/app/new/controllers/person-search-results.js

```diff
@@ -36,7 +36,10 @@ class PersonSearchResultsController extends PersonController {
   setPeopleItems(req, res, next) {
     const { people } = req.form.options.fields
     people.items = req.people.map(person => {
-      const card = presenters.profileToCardComponent({ showTags: false })({
+      const card = presenters.profileToCardComponent({
+        locationType: req.query.filter.prison_number ? 'prison' : 'other',
```

Comment:
Could we use `from_location_type` from the session data here instead, I think that would be safer than manually setting the value.

You can get it using:

```javascript
const { from_location_type: locationType } = req.sessionModel.toJSON()
...
const card = presenters.profileToCardComponent({
  locationType,
  showTags: false,
})({ person })
```